docs/CHANGELOG.asciidoc

```diff
@@ -65,6 +66,318 @@ Review important information about the {kib} 8.x releases.
+
+[float]
+[[fixes-v8.14.0]]
+=== Bug Fixes
+Alerting::
+* Fixes bug with aggregation building for {es} query rule when there are multi-terms and a group by field ({kibana-pull}182865[#182865]).
+* Fixes using `recoveredCurrent` and `activeCurrent` to determine how to update old alerts ({kibana-pull}180934[#180934]).
+* Fixes logging the errors reported by `addLastRunError` to the console ({kibana-pull}179962[#179962]).
+* Preserves relative snooze when adding or removing snooze schedules ({kibana-pull}178344[#178344]).
+* Reverts changes to notify when there is a change on connector configuration ({kibana-pull}177054[#177054]).
+APM::
+* Fixes the cardinality count for SLOs generated from a single SLI definition was previously incorrect for APM latency and APM availability SLIs ({kibana-pull}183171[#183171]).
+* Fixes the telemetry collection of Logstash with metricbeat monitoring ({kibana-pull}182304[#182304]).
+* Fixes otel service detection ({kibana-pull}180574[#180574]).
+Cases::
+* Displays the link to the Cases page under observability when Cases {kib} privileges are granted regardless of the other application privileges ({kibana-pull}182569[#182569]).
+Canvas::
+* Fixes workpad templates using legacy filters function ({kibana-pull}176093[#176093]).
+Connectors::
+* Removes secrets from the connectors before validating in `actionsClient` getAll ({kibana-pull}179837[#179837]).
+Dashboard::
+* Fixes opening panel title edit flyout only when panel title is clicked ({kibana-pull}180137[#180137]).
+* Disallows copy to dashboard from a maximized panel ({kibana-pull}179422[#179422]).
+Discover::
+* Fixes issue where an ES query rule could be created with a data view, then the data view is changed but there's still a reference to the previous data view's timestamp field. ({kibana-pull}182883[#182883]).
+* Fixes view all matches button for timestamps with numeric date formats ({kibana-pull}181769[#181769]).
+* Fixes time range filters for CSV when a relative time filter is specified in UI ({kibana-pull}181067[#181067]).
+* Fixes the status list to be static ({kibana-pull}177435[#177435]).
+* Fixes a timeout for a "View all matches" request ({kibana-pull}181859[#181859]).
+* Fixes tracking total hits for "View all matches" button ({kibana-pull}181811[#181811]).
+* Fixes comments bugs in Discover and Data Visualizer ({kibana-pull}181283[#181283]).
+* Fixes the problem with Discover and queries without the from command ({kibana-pull}180692[#180692]).
+* Fixes displaying "Unsaved changes" badge on time filter changes in case time range is stored along with saved search ({kibana-pull}178659[#178659]).
+Elastic Security::
+For the Elastic Security 8.14.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+Elastic Search::
+* Adds Playground telemetry ({kibana-pull}181587[#181587]).
```

Comment:
nit: generally I don't think we need to document telemetry-related changes in the public release notes

Reply:
Ok I'll remove them, thanks for catching that!